packages/firebase_auth/firebase_auth_web/lib/firebase_auth_web.dart

```diff
@@ -357,4 +357,10 @@ class FirebaseAuthWeb extends FirebaseAuthPlatform {
     // See https://github.com/flutter/flutter/issues/46021
     throw UnimplementedError('verifyPhoneNumber');
   }
+
+  Future<void> confirmPasswordReset(
+      String app, String oobCode, String newPassword) async {
```

Comment:
Adding a trailing comma after `newPassword` may yield a more pleasing formatting (like [this one](https://github.com/FirebaseExtended/flutterfire/blob/master/packages/firebase_auth/firebase_auth_platform_interface/lib/firebase_auth_platform_interface.dart#L217-L223)).